FRAuth/FRAuth/Callbacks/DeviceProfileCallback.swift

```diff
@@ -12,12 +12,16 @@
 import Foundation
 
 /// DeviceAttributeCallback is a callback class that collects Device Information using DeviceCollector(s) in FRAuth SDK.
```

Comment:
The class name was updated to DeviceProfileCallback, should it be reflected in the doc?

Reply:
Corrected with the latest commit.